### PR TITLE
[BUG]fix "Connect to nodejs.org:443"

### LINKED ISSUE #6782: Connect to nodejs.org:443
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description
```
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.15.0:install-node-and-npm (install node and npm) on project seata-console: Could not downloa
d Node.js: Could not download https://nodejs.org/dist/v19.5.0/node-v19.5.0-win-x64.zip: Connect to nodejs.org:443 [nodejs.org/104.20.22.46, nodejs.org/104.20.23.46] failed: Connection timed out: connect -> [Help 1]
```
建议与npm源一样都改为国内源,如下
It is recommended to change to domestic sources like npm sources, as follows
`<nodeDownloadRoot>https://mirrors.aliyun.com/nodejs-release/</nodeDownloadRoot>`

### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
Just paste your stack trace here!
```

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

Minimal yet complete reproducer code (or URL to code):

### Ⅵ. Environment:

- JDK version(e.g. `java -version`):
- Seata client/server version:  
- Database version:
- OS(e.g. `uname -a`):
- Others:

### PR DESCRIPTION
**Pull request by heliang666s**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
add nodeDownloadRoot to fix bug

### Ⅱ. Does this pull request fix one issue?

fixes #6782 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
Unnecessary

### Ⅳ. Describe how to verify it
Unnecessary

### Ⅴ. Special notes for reviews
Unnecessary

### CODE REVIEW

**Comment by xjlgod** [comment]
please add fix info in `changse` module

File: console/pom.xml [hunk 1]
```diff
@@ -36,6 +36,7 @@
         <spring-framework-for-server.version>5.3.37</spring-framework-for-server.version>
         <snakeyaml-for-server.version>2.0</snakeyaml-for-server.version>
         <tomcat-embed.version>9.0.90</tomcat-embed.version>
+        <NODE_DOWNLOAD_URL>https://mirrors.aliyun.com/nodejs-release/</NODE_DOWNLOAD_URL>
```
Comment (slievrly) [inline console/pom.xml:39]: Thank you very much for your PR submission and attention to this issue. However, according to the discussion in the issue, it is not recommended to directly change this place globally, and there may be network problems for foreign users to connect to this address. It is suggested that local issues be solved locally.

Comment (heliang666s) [reply console/pom.xml:39]: So do I need to close this PR？

Comment (slievrly) [reply console/pom.xml:39]: Yes. I look forward to your better fix.

**Pull request closed without merging**